### PR TITLE
Add Crowdin branch parameter for Download and Export

### PR DESCRIPTION
**Pull request by pierallard**
Add missing parameter for new version management
https://crowdin.com/page/api

### CODE REVIEW

File: lib/Crowdin/Api/Download.php [hunk 1]
```diff
@@ -78,4 +80,20 @@ public function getCopyDestination()
     {
         return $this->copyDestination;
     }
+
+    /**
+     * @return string
+     */
+    public function getBranch()
```
Comment (willy-ahva) [inline lib/Crowdin/Api/Download.php]: Why did you add getter setter ?

**Comment by jjanvier** [comment]
GTM ;)

File: lib/Crowdin/Api/Download.php (hunk 1 above)
Comment (pierallard) [inline lib/Crowdin/Api/Download.php]: To get and set ?

**Comment by willy-ahva** [comment]
Nice troll. GTM go set and get your things !

**Comment by solivier** [comment]
gtm but you could remove setter and getters they are not used
